components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
You're member the of WDO (warnings' defenders organization), but I get your point. I think you're right.